.pipelines/networkobservability/pipeline.yaml

```diff
@@ -0,0 +1,145 @@
+pr: none
+trigger: none
+
+schedules:
+- cron: '0 0 * * *'
+  displayName: Daily midnight Cilium with Hubble
+  branches:
+    include:
+    - master
+
+variables:
+  clustername: ciliumhubble-$(Build.SourceBranchName)-$(Build.BuildId)
```

Comment:
How many characters does this normally come out to? 26?

Reply:
In another branch I actually change this format since it gets long. The branch off this branch is cilium-$(Build.BuildId)-$(stack)

Reply:
If I remember right, the magic number to ensure you can use any region in AKS is 28. AKS char limit is 80 for context.

Reply:
I ran into the 80 char limit earlier today which was the main driver to changing the naming strategy.

Reply:
See: https://github.com/Azure/azure-container-networking/pull/2517